src/Controller/ControllerFactory.php

```diff
@@ -108,8 +108,8 @@ public function getControllerClass(ServerRequest $request): ?string
             if ($firstChar !== strtoupper($firstChar)) {
                 deprecationWarning(
                     "The `{$prefix}` prefix did not start with an upper case character. " .
-                    "Routing prefixes should be defined as CamelCase values. " .
-                    "Prefix inflection will be removed in 5.0"
+                    'Routing prefixes should be defined as CamelCase values. ' .
```

Comment:
Not sure how I feel about having to perfectly switch when editing/writing multi-line strings.

Reply:
For such cases sprintf() could be used to avoid using both type of quotes.

Reply:
Sounds reasonable to me.